pkg/config/config_template.yaml

```diff
@@ -3303,3 +3303,15 @@ api_key:
     #
     # span_name_remappings:
     #   <OLD_NAME>: <NEW_NAME>
+
+  ## @param debug - custom object - optional
+  ## Debug-specific configuration for OTLP ingest in the Datadog Agent.
+  #
+  # debug:
+
+    ## @param log_level - string - optional - default: info
```

Comment:
To be consistent with the logging exporter this should be `loglevel` https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter#logging-exporter (sorry to be insistent on this, I think it is very important that we are as consistent as possible to avoid end-user confusion)

Reply:
@mx-psi Thanks for reviewing it! Fixed it.